pages/landing/enterprise-eks/request-access/index.html

```diff
@@ -0,0 +1,105 @@
+        available: now
+        plans: ["Standard", "Early Access"]
+      - name: CIS Benchmark
+        description: Optionally, entirely Amazon EKS clusters that have been hardened according to the CIS Benchmark for Kubernetes and CIS Benchmark for Amazon EKS, all defined as Terraform code.
+        plans: ["Standard", "Early Access"]
+      - name: Istio
+        description: Use the Istio service mesh to enable service-to-service communication with security, flexibility, and observability.
+        plans: ["Standard", "Early Access"]
+      - name: AWS App Mesh
+        description: Set up AWS App Mesh to monitor your applications on the network, control traffic, and connect microservices, with features for automatic encryption, retries, and more.
+        plans: ["Standard", "Early Access"]
+      - name: Consul Connect
+        description: Configure HashiCorp Consul Connect for automatic inter-service encryption, identity-based authorization, and traffic observability.
+        plans: ["Standard", "Early Access"]
+      - name: Flux
+        description: Deploy your apps using GitOps practices with Flux. Employ techniques such as canary deployments, feature flags, and staggered deployments, with built-in notifications and alerts.
+        plans: ["Standard", "Early Access"]
+      - name: Argo
+        description: Achieve continuous delivery with Argo CD. Argo includes automatic deployments for a variety of standard application definition options, such as Kustomize, Helm, and more. Manage multiple clusters and integrate with a variety of managed and self-hosted version control systems.
+        plans: ["Standard", "Early Access"]
+      - name: Prometheus
+        description: Instrument your apps and ship metrics to Prometheus, the monitoring and alerting toolkit. Query metrics along multiple dimensions and create dashboards to view app behavior. 
+        plans: ["Standard", "Early Access"]
+      - name: Datadog
+        description: If you prefer hosted services, ship logs and metrics to Datadog and benefit from the native Kubernetes and EKS integration.
+        plans: ["Standard", "Early Access"]
+      - name: Grafana
+        description: Create beautiful, intuitive dashboards to track your application performance using Grafana, the open observability platform.
+        plans: ["Standard", "Early Access"]
+      - name: Lens
+        description: Improve your productivity with Amazon EKS using Lens, the Kubernetes IDE, a graphical web UI that makes using Kubernetes easier and more productive.
+        plans: ["Standard", "Early Access"]
+      - name: Porter
+        description: Combine the benefits of Kubernetes with the ease of Heroku using Porter, a Platform-as-a-Service for your Amazon EKS clusters.
+        plans: ["Standard", "Early Access"]
```

Comment:
FYI I'm validating these two techs now to see if it makes sense to mention it on the landing page. After reading some more about k8s Lens, I'm beginning to doubt if it makes sense to mention it as something we support, as now I believe it's actually a local app that you run on the operator machine with nothing on the server side.

It appears to be a tool that uses the base Kubernetes API to get the information it needs, similar to how `kubectl` works.

Reply:
Confirmed that `Lens` is a local IDE that doesn't need anything in the cluster. We should remove it from the LP since it doesn't make sense in the context of our offering (server side EKS features).

Reply:
I also started looking into `porter`, and I'm not sure this is also something we can support. Porter appears to be a SaaS app that requires full control over the EKS cluster (that is, porter needs to deploy and manage the EKS cluster to get all the benefits - [see their getting started on AWS guide to get what I mean](https://docs.porter.run/docs/getting-started-with-porter-on-aws))

Reply:
I'll spend some time tomorrow to get some signals on what folks are using for server side dashboards, or if it is completely old school given the powerful local side UI dashboards like `Lens` and `Infra App`.

Reply:
I removed Lens in the latest commit. Feel free to add and remove features from the list as needed.

Reply:
One thought: even if Lens is purely a local IDE, it can still be a first-class part of the experience we provide customers. E.g., We can have some sort of guided experience for app devs of (a) how to package / instrument their apps and (b) how to take those packaged apps and deploy them using Lens. That experience may be a wizard in a web UI, or docs/tutorials, or some CLI experience. But the point is to show that, for app devs, a UI is central to what we provide, and makes it 10x easier to get started with K8S.

Reply:
I think there is a difference between marketing something, and having support for it. To me, offering Lens on the marketing page is like offering VSCode for developers. It makes us look like we don't know what we are talking about.

I'm not saying there isn't anything we can do with Lens here. What I am saying is that it doesn't really look good to us to have it on the marketing page as a supported feature. I think the following thought process would be very common amongst users: "Oh I never heard of Lens before - let me investigate what it is. Oh it's a local GUI app. Is there a server side component? Wait... there's no server side component? Then what does Gruntwork offer to support it? Do these guys even know what Lens is? Hmm I'm not sure I trust them anymore..." And then for those that actually know what Lens is, it would be even more severe.

> But the point is to show that, for app devs, a UI is central to what we provide, and makes it 10x easier to get started with K8S.

I agree, and to be clear, we're not completely abandoning this angle. It's just that we need to phrase this section/component differently than "we offer support for Lens."

Reply:
Ok I am going 360 on this (two 180 turns) and added back in Lens, but under self service deployments. I am still on the fence if it makes sense to include it, but after reviewing the landscape, there really isn't much in terms of server side self service that makes sense for us to offer (e.g., Porter is a SaaS product that needs control over the EKS cluster, which is against the spirit of "have access to 100% of the code". Rancher is slightly better, but it's questionable if we can make a Rancher offering based on [their EULA](https://rancher.com/eula/)).

FWIW, my favorite after surveying a bunch of tools and configurations is actually Argo CD. Argo CD has a pretty slick UI baked into their deployments system that provides key deployment rollout statuses and debug logs. See [their video demo here](https://www.youtube.com/watch?v=aWDIQMbp1cc&t=64s).

Reply:
I have no strong opinions on _which_ UI (I'd have to try them!), so long as we call out that part of the workflow is a first-class UI. Perhaps we even list _options_ (e.g., Argo CD, as you said) and let users guide us later. I will say that Argo seems to have a _lot_ of powerful K8S functionality available (e.g., Argo Rollouts for blue green deploys, Argo CD for UI), so perhaps that's a good angle to dig into more at some point.

Reply:
Looking at the latest UI, and I think you did just that in the YAML file. Nice!